utils/json_spec.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+//import "encoding/json"
+import "github.com/goccy/go-json"
+
+func JSONUnmarshal(data []byte, v any) error {
```

Comment:
rename to JsonUnmarshal / JsonMarshal is more readable and look better.

Reply:
fixed